.github/workflows/ci-tests.yaml

```diff
@@ -117,3 +117,27 @@ jobs:
       - name: Run manifest tests
         run: |
           make test-manifests
+
+  registry-scanner:
+    name: Ensure registry-scanner Go modules synchronicity and run tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Setup Golang
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+      - name: Download all Go modules
+        run: |
+          go mod download
```

Comment:
do we need to first `cd registry-scanner` here too?

Reply:
Good point. Moved the cd here.